### ISSUE TITLE
Unable to move Kap palette away from main display monitor

### ISSUE THREAD

**Issue by steveharman** [wulkano/kap#551]
macOS version:
ProductName:	Mac OS X
ProductVersion:	10.13.4
BuildVersion:	17E199

Kap version:
3.0.0 Beta 1.

#### Steps to reproduce
Access Kap from the menu bar

#### Current behaviour
The floating "palette" can't be moved off the main monitor to a secondary or tertiary screen, in a multi-display system. 

#### Expected behaviour
 I have three screens and when using Kap would like to move the palette out of the way, from my main display on to one of my of my secondary screens.

#### Workaround
Unknown.

**Comment by skllcrn** [comment]
By "palette" I'm presuming you mean the main Kap toolbar @steveharman? 

![image](https://user-images.githubusercontent.com/8440026/46198900-d3311300-c30d-11e8-8dc3-0f3e82fef62e.png)

**Comment by steveharman** [comment]
That's the one, yes @skllcrn   Wasn't sure of the terminology.

**Comment by karaggeorge** [comment]
You should be able to start drawing a cropper on a different display and the bat will follow there. You can’t manually move it but it follows the cropper

**Comment by steveharman** [comment]
Understood, thanks @karaggeorge - my initial reaction and perhaps that of others would be to drag the toolbar "out of the way" (off my main monitor to a side-screen) and then begin drawing.

Plus I can't think of another tool where I'm prevented from dragging its "windows" from screen to screen.  

But as you say, draw on a second screen and the toolbar comes along.  It's more the minor usability niggle of being prevented from dragging the toolbar form screen to screen.

Cheers

**Comment by karaggeorge** [comment]
@steveharman Yeah, understood. The "problem" is that the bar is not really a window. Each screen has it's own window with the overlay, cropper and bar being html components. So dragging the bar doesn't really work from window to window. 

But honestly it seems easier to me to just move your cursor two screens down and have it follow you automatically instead of having to click + drag it over. 

That said, maybe we need a better way of letting users know that's how it works

**Comment by steveharman** [comment]
Agreed. @karaggeorge this was more about confusion over use on my part.  As I didn't need to click / draw a cropper on a second screen it didn't cross my mind to venture on to another screen for any reason or assume the toolbar would follow. 

My case was wanting to draw on my main screen so I thought I'd clear the toolbar out of the way first  [to a different screen] and found I couldn't.  

Anyway, really pleased to have the floating toolbar back!  :-)

**Comment by skllcrn** [comment]
I think the best solution to this is to be able to drag the main toolbar between screens. However, for people who don't attempt to to drag the toolbar to a custom position it's nice to be able to click on the screen you're wanting to work on to have the toolbar spawn there.

**Comment by skllcrn** [comment]
This isn't feasible right now, putting the issue on hold.
